[gen-disconnect-called-during-success-with-uuid.https.html]
  [disconnect() called during a getPrimaryServices call that succeeds. Reject with NetworkError.]
    expected: FAIL

